hathor/p2p/protocol.py

```diff
@@ -74,12 +74,12 @@ class WarningFlags(str, Enum):
         NO_ENTRYPOINTS = 'no_entrypoints'
 
     network: str
-    my_peer: Peer
+    my_peer: PrivatePeer
     connections: 'ConnectionsManager'
     node: 'HathorManager'
     app_version: str
     last_message: float
-    peer: Optional[Peer]
+    peer: Optional[PublicPeer]
```

Comment:
Maybe we should do the `_peer` with `@property peer(): assert self._peer is not None; return _peer` since it cannot be `None` for most of our codebase.

Reply:
Done.

Reply:
Thanks!